www/src/App.test.tsx

```diff
@@ -16,9 +29,60 @@ describe('App', () => {
         ${'interviewer'} | ${'🚧 Work in progress 🚧'} | ${'an interviewer'}
         ${'admin'}       | ${'🚧 Work in progress 🚧'} | ${'an admin'}
     `('renders correctly for $friendlyName', ({ role, containsString }) => {
-        render(withAuth0(withRootState(<App />, { user: { roles: [], currentRole: role } }), { isAuthenticated: true }));
+        const dispatchMock = jest.fn();
+        useAppDispatchMock.mockReturnValue(dispatchMock);
+
+        const rootState = { user: { roles: [], currentRole: role, isLoading: false, hasRegistered: true } };
+        const auth0State = { isAuthenticated: true };
+
+        useAppSelectorMock.mockImplementation((selector) => selector(rootState));
```

Comment:
I needed to add these because we're now mocking the entire `./redux/hooks` module